pkg/cmd/init/cmd.go

```diff
@@ -74,9 +74,11 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 		"The credential file is the docker config json file and will be filled into the default image pull secret named open-cluster-management-image-pull-credentials.")
 	cmd.Flags().StringVar(&o.bundleVersion, "bundle-version", "default",
```

Comment:
Is the `bundle-version` only useful for setting things to `latest`? For example, if I choose v0.16.1, it also sets other components (multicluster_controlplane, etc.) to v0.16.1, even though that version might not exist for those components.